source/containers.tex

```diff
@@ -73,12 +73,20 @@
 In Tables~\ref{tab:container.req},
 \ref{tab:container.rev.req}, and
 \ref{tab:container.opt}
```

Comment:
We sometimes have a comma after "In tables A, B, C\opt{,}" and sometimes not. I think with the new list format, a comma is always good, what do you think?

Reply:
Made it a comma everywhere (the unordered container requirements had a colon).

Reply:
Thanks!